### PR TITLE
[APT-1602] Import Landing zone guide

### PR DESCRIPTION
**Pull request by oredavids**
[APT-1602] 

[APT-1602]: https://gruntwork.atlassian.net/browse/APT-1602?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/intro/zero2hero/html-content.mdx [hunk 1]
```diff
@@ -3,3 +3,8 @@ import AsciiDocContent from "/src/components/AsciiDocContent";
 # Rendering HTML content
 
 <AsciiDocContent content="fooo" />
+
```
Comment (eak12913) [inline docs/intro/zero2hero/html-content.mdx:6]: You checked this in by accident. I think we will delete this page in the "base" PR. So can you please revert this. I will go check as I may have _also_ checked this in by accident.

File: _docs-sources/guides/build-it-yourself/0-landing-zone/0-intro/0-what-youll-learn-in-this-guide.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+<div className="dlist">
+
+#### [Core concepts](../1-core-concepts/0-aws-account.md)
+
+An overview of the core concepts you need to understand to set up an AWS account structure, including AWS
+Organizations, IAM Users, IAM Roles, IAM Groups, CloudTrail, and more.
+
+#### [Production-grade design](../2-production-grade-design/1-the-root-account.md)
```
Comment (eak12913) [inline _docs-sources/guides/build-it-yourself/0-landing-zone/0-intro/0-what-youll-learn-in-this-guide.md:19]: Should this link to not go the `intro` section?

File: docs/intro/zero2hero/html-content.mdx (hunk 1 above)
Comment (oredavids) [reply docs/intro/zero2hero/html-content.mdx]: ✅ Pulled base branch.

File: _docs-sources/guides/build-it-yourself/0-landing-zone/0-intro/0-what-youll-learn-in-this-guide.md (hunk 2 above)
Comment (oredavids) [reply _docs-sources/guides/build-it-yourself/0-landing-zone/0-intro/0-what-youll-learn-in-this-guide.md]: ✅ Good catch.

**Pull request merged**